src/Test/Html/Query.elm

```diff
@@ -134,6 +135,42 @@ findAll selectors (Internal.Single showTrace query) =
         |> Internal.Multiple showTrace
 
 
+{-| Find the descendant elements of the result of `findAll` which match all the given selectors.
+
+    import Html exposing (div, ul, li)
+    import Html.Attributes exposing (class)
+    import Test.Html.Query as Query
+    import Test exposing (test)
+    import Test.Html.Selector exposing (tag)
+    import Expect
+
+
+    test "The list has three items" <|
+        \() ->
+            div []
+                [ ul [ class "items active" ]
+                    [ li [] [ a [] [ text "first item" ]]
+                    , li [] [ a [] [ text "second item" ]]
+                    , li [] [ a [] [ text "third item" ]]
+                    , li [] [ button [] [ text "button" ]]
+                    ]
+                ]
+                |> Query.fromHtml
+                |> Query.findAll [ tag "li" ]
+                |> Query.filterMap [ tag "a" ]
+                |> Expect.all
+                    [ Query.each (Query.has [ tag "a" ])
+                    , Query.first >> Query.has [ text "first item" ]
+                    ]
+
+-}
+filterMap : List Selector -> Multiple msg -> Multiple msg
```

Comment:
The type of `filterMap` is suspicious to me: I expect it to be like: `(Selector -> Maybe a) -> Multiple msg -> Multiple msg`. It can't be `Selector -> Maybe a`, but it can be singular instead of a list. What's the benefit of having a list? e.g I don't know what `filterMap [ tag "a", class "fish" ]` should do